mypy/config_parser.py

```diff
@@ -109,6 +109,9 @@ def split_and_match_files_list(paths: Sequence[str]) -> list[str]:
     expanded_paths = []
 
     for path in paths:
+        if not path:
+            continue
```

Comment:
Is this necessary, provided that the input validation would take care of violations and normalize the paths?